sql/updates/world/master/2024_99_99_99_shadow_focus_and_premeditation.sql

```diff
@@ -0,0 +1,9 @@
+DELETE FROM `spell_proc` WHERE `SpellId` IN (343173);
+INSERT INTO `spell_proc` (`SpellId`,`SchoolMask`,`SpellFamilyName`,`SpellFamilyMask0`,`SpellFamilyMask1`,`SpellFamilyMask2`,`SpellFamilyMask3`,`ProcFlags`,`ProcFlags2`,`SpellTypeMask`,`SpellPhaseMask`,`HitMask`,`AttributesMask`,`DisableEffectsMask`,`ProcsPerMinute`,`Chance`,`Cooldown`,`Charges`) VALUES
+(343173,0x00,8,0x00000000,0x00400000,0x00000000,0x00000000,0x0,0x0,0x1,0x2,0x0,0x0,0x0,0,0,0,0); -- Premeditation
```

Comment:
This mask includes too many spells (finishing moves, instant poison damage proc)